src/reducers/userprofile.js

```diff
@@ -12,13 +12,6 @@ const initialState = {
             errorMessage: null,
             userprofile: action.payload
         };
-        case 'USER_PROFILE_EDIT':
-        return {
-            ...state,
-            error: false,
-            errorMessage: null,
-            userprofile: action.payload
-        };
         case 'USER_PROFILE_UPDATED':
         return {
             ...state,
```

Comment:
dont copy all state if you rewrite every field of it

Reply:
ok